### PR TITLE
[MIG] stock_landed_costs_currency: Migration to 16.0 

### PR DESCRIPTION
**Pull request by BT-jdziurzynski**

### CODE REVIEW

File: stock_landed_costs_currency/models/stock_landed_cost.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+
+class LandedCost(models.Model):
+    _inherit = "stock.landed.cost"
+
+    currency_id = fields.Many2one(
+        comodel_name="res.currency",
+        required=True,
+        related=None,
+        default=lambda self: self.env.user.company_id.currency_id,
```
Comment (BT-anieto) [inline stock_landed_costs_currency/models/stock_landed_cost.py]: Get the company from the environment like self.env.company.currency_id

**Review by BT-anieto — COMMENTED** [review]
Hi @BT-jdziurzynski! Thanks for your PR. Just a small comment :)

**Review by BT-anieto — APPROVED** [review]

**Review by BT-aleonard — APPROVED** [review]
Hi, thanks for migrating. 
The only thing I would add is to add yourself as a contributor.

**Review by BT-dmoreno — APPROVED** [review]

**Comment by BT-jdziurzynski** [comment]
Hey @rousseldenis, I don't understand why codecov check is reduced and not passing after I added myself as a contributor. Could you take a look at this?

**Comment by rousseldenis** [comment]
> Hey @rousseldenis, I don't understand why codecov check is reduced and not passing after I added myself as a contributor. Could you take a look at this?

@BT-jdziurzynski You can always inspect the lines that are missing in tests if you go to the 'Files changed' tab here.

image


So, adding a test that cover that line should greenify coverage.

For contributor problem, I don't get what you mean.

**Comment by BT-jdziurzynski** [comment]
> > Hey @rousseldenis, I don't understand why codecov check is reduced and not passing after I added myself as a contributor. Could you take a look at this?
> 
> @BT-jdziurzynski You can always inspect the lines that are missing in tests if you go to the 'Files changed' tab here.
> 
> image
> 
> So, adding a test that cover that line should greenify coverage.
> 
> For contributor problem, I don't get what you mean.

I already extended the tests to cover these lines and codecov was satisfied. This changed after I added myself as a contributor.

**Comment by rousseldenis** [comment]
> > > Hey @rousseldenis, I don't understand why codecov check is reduced and not passing after I added myself as a contributor. Could you take a look at this?
> > 
> > 
> > @BT-jdziurzynski You can always inspect the lines that are missing in tests if you go to the 'Files changed' tab here.
> > image
> > So, adding a test that cover that line should greenify coverage.
> > For contributor problem, I don't get what you mean.
> 
> I already extended the tests to cover these lines and codecov was satisfied. This changed after I added myself as a contributor.

@BT-jdziurzynski Apparently not as this is the result for last build

**Comment by BT-anieto** [comment]
Hello @rousseldenis Tests are passing right now. Could it be that the needs_review tag is preventing the PR to be merged?
Shall anything else be done here? Thank you very much!

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**